source/tyr/tyr/__init__.py

```diff
@@ -35,11 +35,16 @@
 from redis import Redis
 from celery.signals import setup_logging
 from flask_script import Manager
+from flask_cors import CORS
 from tyr.rabbit_mq_handler import RabbitMqHandler
 
 app = Flask(__name__)
 app.config.from_object('tyr.default_settings')  # type: ignore
 app.config.from_envvar('TYR_CONFIG_FILE', silent=True)  # type: ignore
+
+CORS(app)
```

Comment:
Sonarcloud [flags](https://sonarcloud.io/project/security_hotspots?id=Hove_navitia&pullRequest=3888&resolved=false&types=SECURITY_HOTSPOT) this as a security risk.
It suggests to allow CORS only for specific origins.
So maybe something like : 
```suggestion
CORS(app, origins=TYR_TRUSTED_ORIGIN)
```
where `TYR_TRUSTED_ORIGIN` is configured to be something like `https://navitia-admin.archi.sbx.aws.private` like specified in https://navitia.atlassian.net/browse/NAV-1634 ?

Reply:
Revert